javascript/src/signals.ts

```diff
@@ -77,7 +97,8 @@ export function computeS(r: Uint8Array, secretKey: Uint8Array, c: string) {
 export function computeAllInputs(
   message: string | Uint8Array,
   secretKey: string | Uint8Array,
-  r?: string | Uint8Array
+  r?: string | Uint8Array,
+  version: PlumeVersion = PlumeVersion.V1
```

Comment:
confirming we want the default version to be v1?

Reply:
I think this can be a latest version(V2). But I don't want to modify a old tests, so I keep it V1.